### DISCUSSION TITLE
Rotate the board

### DISCUSSION THREAD

**Discussion by micha-uhl** [nicolacimmino/PLC-14500#11] [Show and tell]
The board should rotate 180 degrees, so the LEDs count from right (bit 0) to left!

![IMG_3336](https://github.com/nicolacimmino/PLC-14500/assets/118008310/a9adce1a-b963-47f9-b0cd-9a50a3337def)

I used LEDs in different colors for better overview. Nice project!

**Comment by nicolacimmino** [comment]
Thanks!

Yep, I got that comment few times. I currently have a batch of boards still sitting under my table, but once I run out I will change the LEDs order. 

Out of curiosity, C7 isn't the one I ship. Was it faulty or did you forget to install it before the Arduino and you had to rip it out to rework it? ;)

-Nicola

**Comment by micha-uhl** [comment]
Hallo Nicola,
I have changed C7 in 220uF and C10 in 100uF for my preferred clock rate. I don’t need full speed, it’s more for running lights.
Michael

> Am 11.03.2024 um 13:34 schrieb Nicola Cimmino ***@***.***>:
> 
> Yep, I got that comment few times. I currently have a batch of boards still sitting under my table, but once I run out I will change the LEDs order.
> 
> Out of curiosity, C7 isn't the one I ship. Was it faulty or did you forget to install it before the Arduino and you had to rip it out to rework it? ;)
> 
>
